docs/source/en/model_doc/vitpose.md

```diff
@@ -10,277 +10,107 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
-
-- 0: [COCO validation 2017](https://cocodataset.org/#overview) dataset, using an object detector that gets 56 AP on the "person" class
-- 1: [AiC](https://github.com/fabbrimatteo/AiC-Dataset) dataset
-- 2: [MPII](https://www.mpi-inf.mpg.de/departments/computer-vision-and-machine-learning/software-and-datasets/mpii-human-pose-dataset) dataset
-- 3: [AP-10K](https://github.com/AlexTheBad/AP-10K) dataset
-- 4: [APT-36K](https://github.com/pandorgan/APT-36K) dataset
-- 5: [COCO-WholeBody](https://github.com/jin-s13/COCO-WholeBody) dataset
-
-Pass the `dataset_index` argument in the forward of the model to indicate which experts to use for each example in the batch. Example usage is shown below:
+</hfoption>
+<hfoption id="AutoModel">
 
 ```python
-image_processor = AutoProcessor.from_pretrained("usyd-community/vitpose-plus-base")
-model = VitPoseForPoseEstimation.from_pretrained("usyd-community/vitpose-plus-base", device=device)
-
-inputs = image_processor(image, boxes=[person_boxes], return_tensors="pt").to(device)
+import torch
+from PIL import Image
+from transformers import AutoProcessor, VitPoseForPoseEstimation
 
-dataset_index = torch.tensor([0], device=device) # must be a tensor of shape (batch_size,)
+image = Image.open("path/to/image.jpg")
+processor = AutoProcessor.from_pretrained("usyd-community/vitpose-base-simple")
+model = VitPoseForPoseEstimation.from_pretrained("usyd-community/vitpose-base-simple")
 
+inputs = processor(image, return_tensors="pt")
 with torch.no_grad():
-    outputs = model(**inputs, dataset_index=dataset_index)
+    outputs = model(**inputs)
 ```
 
-The ViTPose+ checkpoints use 6 experts, hence 6 different dataset indices can be passed. 
-An overview of the various dataset indices is provided below:
-
-- 0: [COCO validation 2017](https://cocodataset.org/#overview) dataset, using an object detector that gets 56 AP on the "person" class
-- 1: [AiC](https://github.com/fabbrimatteo/AiC-Dataset) dataset
-- 2: [MPII](https://www.mpi-inf.mpg.de/departments/computer-vision-and-machine-learning/software-and-datasets/mpii-human-pose-dataset) dataset
-- 3: [AP-10K](https://github.com/AlexTheBad/AP-10K) dataset
-- 4: [APT-36K](https://github.com/pandorgan/APT-36K) dataset
-- 5: [COCO-WholeBody](https://github.com/jin-s13/COCO-WholeBody) dataset
+</hfoption>
+<hfoption id="transformers-cli">
 
+```bash
+transformers-cli run --task pose-estimation --model usyd-community/vitpose-base-simple --image path/to/image.jpg
+```
 
-### Visualization
-
-To visualize the various keypoints, one can either leverage the `supervision` [library](https://github.com/roboflow/supervision (requires `pip install supervision`):
+</hfoption>
+</hfoptions>
 
-```python
-import supervision as sv
+---
 
-xy = torch.stack([pose_result['keypoints'] for pose_result in image_pose_result]).cpu().numpy()
-scores = torch.stack([pose_result['scores'] for pose_result in image_pose_result]).cpu().numpy()
+## Quantization Example
```

Comment:
Probably better to use a different quantization backend, like torchao, that isn't specific to Intel hardware. Remember to add the complete code snippet for using quantization!

```suggestion
Quantization reduces the memory burden of large models by representing the weights in a lower precision. Refer to the [Quantization](https://huggingface.co/docs/transformers/main/en/quantization/overview) overview for more available quantization backends.

The example below uses [torchao](https://huggingface.co/docs/transformers/main/en/quantization/torchao) to only quantize the weights to int4.
```

Reply:
Remove `## Quantization Example` and add the text above!